guide/english/r/objects/index.md

```diff
@@ -36,19 +36,29 @@ The following creates the function circleArea that calculates the area of a circ
 1. To create an R object, choose a name and then use the less-than symbol, `<`,
 followed by a minus sign,  `-`, to save data into it. This combination looks like an
 arrow, `<-`. R will make an object, give it your name, and store in it whatever
-follows the arrow. You can also use the '=' sign, but this is a less common method to assign a value to an object and generally frowned upon.
+follows the arrow. You can also use the '=' sign, but this is a less common method to assign a value to an object and generally frowned upon. There is one function named `assign` in R, which can also be used as an assignment operator. 
 
 2. When the user calls the object in the console it provides the output on the next line. For example:
 
 ```r
 > die <- 1:6
 > die
 [1] 1 2 3 4 5 6
+
+# Change in the direction of arrow
+> 1:6 -> die
+> die
+[1] 1 2 3 4 5 6
+
+
+# Use of assign operator 
+> assign("die", 1:6)
+> die
+[1] 1 2 3 4 5 6
 ```
 
 3. You can name an object in R almost anything you want, but there are a few rules. First,
-a name cannot start with a number. 
-Second, a name cannot use some special symbols, like  `^,  !,  $,  @,  +,  -,  /, or  *`:
+a name cannot start with a number. Second, a name cannot use some special symbols, like  `^,  !,  $,  @,  +,  -,  /, or  *`:
```

Comment:
I would also suggest the use of ordered list in this part of the description, this will provide more emphasis onto the points from the reader.

(However I feel that this could be beyond the scope of this PR, but going through it I understand that we are adding more readability to this article and this addition would make it even better.)